qt/paraview_ext/VatesAPI/src/MDHWInMemoryLoadingPresenter.cpp

```diff
@@ -9,7 +9,6 @@
 #include "MantidVatesAPI/WorkspaceProvider.h"
 #include "MantidVatesAPI/vtkDataSetFactory.h"
 
-#include "tbb/tbb.h"
```

Comment:
I wonder why tbb isn't setting `__TBB_CPP11_TYPE_PROPERTIES_PRESENT` to true, creating issues with GCC 7.

Since removing this header fixes the build issue, it could me moved to a separate issue.